src/ui/utils/LedgerContext.js

```diff
@@ -31,8 +33,21 @@ export const LedgerProvider = ({
   const updateLedger = (_?: Ledger) =>
     setLedgerState({ledger: ledgerManager.ledger});
 
+  const saveToDisk = async () => {
+    try {
+      const res = await ledgerManager.persist();
+      if (res.error) {
+        alert(`Failed to save ledger: ${res.error}`);
+        return;
+      }
+      alert("Ledger saved to disk");
```

Comment:
Have you run this by Faruk?

Reply:
No but I'm not going to focus design work on the outdated explorer / admin UI anymore; instead I'm focusing on ensuring the UI and IXD is designed and UX-optimized for the new Explorer. (Thanks Thena! <3 )